lib/content-services/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -65,34 +68,70 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
     @Input()
     displayAspect: string = null;
 
+    @Input()
+    displayErrors = true;
+
     basicProperties$: Observable<CardViewItem[]>;
     groupedProperties$: Observable<CardViewGroup[]>;
-    disposableNodeUpdate: Subscription;
 
     constructor(
         private contentMetadataService: ContentMetadataService,
         private cardViewUpdateService: CardViewUpdateService,
         private nodesApiService: NodesApiService,
         private logService: LogService,
-        private alfrescoApiService: AlfrescoApiService
+        private alfrescoApiService: AlfrescoApiService,
+        private notificationService: NotificationService,
```

Comment:
didn't we decide to not use anymore the NotificationService is the component but wires it in the apps?